uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestRightsTest.php

```diff
@@ -220,6 +221,66 @@ public function testCanChangeAbsenceTypeForReturnsFalseWhenCurrentUserNotLeaveCo
     );
   }
 
+  public function testCanCancelToilWithPastDatesReturnsTrueWhenCurrentUserIsManagerAndAbsenceTypeDoesNotAllowPastAccrual() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'allow_accruals_request' => true,
```

Comment:
I think most of the booleans in this file are lower case (UNFORGIVABLE) 

Can you run civilint on it?